Source/msg.cpp

```diff
@@ -266,7 +266,7 @@ static BYTE *DeltaExportJunk(BYTE *dst)
 	}
 
 	for (i = 0, q = 0; i < MAXQUESTS; i++) {
-		if (questlist[i]._qflags & QUEST_ANY) {

```

Comment:
So this is not a bitmask anymore, I guess it never was?

Reply:
yeah it only checks if it's only for single player, might as well change it to a bool for what it's worth.